src/installation/guides/macbook.md

```diff
@@ -0,0 +1,53 @@
+# Macbookpro 12,1 (early 2015) 
+
+Everything is well supported on this laptop and install will work without problems. Post-installation, after having booted to your newly installed system, there is however a few points that need a little bit of work.
+
+## keymap are wrong
+This is a problem inherited by a few distros, including archlinux.
+tty keymaps for the macbooks (mac-fr, mac-us, mac-de) are completely broken and *you should not attempt to use them*, or it will result in an unusable keyboard and you will be forced to do a hard reset on your system. as a workaround generic keymaps
+work well (fr, us, de ..) you can use those for starter. later on you can find working ones and install/use them. here i've found a working one for the french macbook keyboards: [forums.archlinux.fr](https://forums.archlinux.fr/viewtopic.php?t=8679)
+
+```
+cd /usr/share/kbd/keymaps/mac/all
+wget ftp://ftp.linux-france.org/pub/macintosh/mac-fr-ext_new.map.gz
+localectl set-keymap mac-fr-ext_new
```

Comment:
I don't belive localectl is available on Void. This is part of systemd, which Void doesn't currently provide.

Reply:
yes, i'll modify that it's not localectl but instead: loadkeys.
i mixed it, sorry for the confusion.